revolv/project/fixtures/category.json

```diff
@@ -0,0 +1,44 @@
+[
+{
```

Comment:
I think loading category code through fixtures is fine for now, as long as we're not loading fixtures as part of setup for tests. Really fixtures are hard to maintain, so we should just move to having some kind of db seed task instead, but I think it's okay for now. Also revolv++